scripts/trilinos-integration/white_run_jenkins_script_cuda-pascal

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -el
+ulimit -c 0
+
+KOKKOSKERNELS_BRANCH=$1
+TRILINOS_UPDATE_BRANCH=$2
+TRILINOS_PRISTINE_BRANCH=$3
+KOKKOS_UPDATE_BRANCH=$4
+
+if [ -z $KOKKOSKERNELS_BRANCH ]
+then
+  KOKKOSKERNELS_BRANCH=develop
+fi
+
+if [ -z $TRILINOS_UPDATE_BRANCH ]
+then
+  TRILINOS_UPDATE_BRANCH=develop
+fi
+
+if [ -z $TRILINOS_PRISTINE_BRANCH ]
+then
+  TRILINOS_PRISTINE_BRANCH=develop
+fi
+
+module load devpack/20180521/openmpi/2.1.2/gcc/7.2.0/cuda/9.2.88
+module swap openblas/0.2.20/gcc/7.2.0 netlib/3.8.0/gcc/7.2.0
```

Comment:
any reason not to use openblas ?

Reply:
There is a Trilinos issue where use of OpenBLAS resulted in lots of problems on White/Ride and the solution they decided upon was to use Netlib instead. I just followed the same course as the ATDM testing.